web/src/testgrid-grid-row-header.ts

```diff
@@ -0,0 +1,32 @@
+import { LitElement, html, css } from "lit";
```

Comment:
IMO `testgrid-grid-header-row` and `testgrid-grid-row-header` are a little bit confusing. perhaps changing the naming would help with that.

how about `testgrid-grid-row-name` or `row-id`?